src/pn_expressions/mode_weights_Psi_M.jl

```diff
@@ -0,0 +1,198 @@
+                         + 69527951ν^2/166053888
+                         + 1321981ν^3/5930496)
+        )
+    end
+
+    # ell=3
+    if ℓₘₐₓ ≥ 3
+        Ψ_M[Yindex(3,1,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1k) of MSB+ (2022)
+            (v/c)^6 * (484δ*ν/(15√21))
+        )
+        Ψ_M[Yindex(3,3,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1i) of MSB+ (2022)
+            (v/c)^6 * (-44δ*ν/(27√35))
+        )
+    end
+
+    # ell=4
+    if ℓₘₐₓ ≥ 4
+        Ψ_M[Yindex(4,0,ℓₘᵢₙ)] = Ψ_M₀ * (1//42) * @pn_expansion(
+            # Eq. (A1q) of MSB+ (2022)
+            1
+            + (v/c)^2 * (-180101//29568 + 27227ν/1056)
+            + (v/c)^4 * (2201411267//158505984
+                         - 34829479ν/432432 + 844951ν^2/27456)
+            + (v/c)^5 * (-13565/1232 + 13565ν/308)π
+            + (v/c)^6 * (15240463356751//781117489152
+                         - (1029744557245//27897053184 + 205π^2/96)*ν
+                         - 4174614175ν^2/36900864
+                         + 221405645ν^3/11860992)
+        )
+        Ψ_M[Yindex(4,4,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1m) of MSB+ (2022)
+            (v/c)^5 * (-4𝒾//3 * √(2//35) * ν)
+        )
+    end
+
+    # ell=5
+    if ℓₘₐₓ ≥ 5
+        Ψ_M[Yindex(5,1,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1w) of MSB+ (2022)
+            (v/c)^6 * (52//21*√(2//165)*δ*ν)
+        )
+        Ψ_M[Yindex(5,3,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1u) of MSB+ (2022)
+            (v/c)^6 * (4δ*ν/(27√385))
+        )
+        Ψ_M[Yindex(5,5,ℓₘᵢₙ)] = Ψ_M₀ * @pn_expansion(
+            # Eq. (A1s) of MSB+ (2022)
+            (v/c)^6 * (-36δ*ν/(5√77))
+        )
+    end
+
+    # ell=6
+    if ℓₘₐₓ ≥ 6
+        Ψ_M[Yindex(6,0,ℓₘᵢₙ)] = Ψ_M₀ * (-4195/(177408√13)) * @pn_expansion(
+            # Eq. (A1y) of MSB+ (2022)
+            + (v/c)^2 * (1 - 3612ν/839)
+            + (v/c)^4 * (-45661561//6342840 + 101414ν/2517 - 48118ν^2/839)
+            + (v/c)^5 * (1248//839 - 4992ν/839)π
+            + (v/c)^6 * (3012132889099//144921208320
+                         - 27653500031ν/191694720
+                         + 1317967427ν^2/4107744
+                         - 24793657ν^3/342312)
+        )
+    end
+
+    # ell=7
+    # Nothing so far
+
+    # ell=8
+    if ℓₘₐₓ ≥ 8
+        Ψ_M[Yindex(8,0,ℓₘᵢₙ)] = Ψ_M₀ * (75601/(8895744√17)) * @pn_expansion(
+            # Eq. (A1aa) of MSB+ (2022)
+            + (v/c)^4 * (1 - 452070ν/75601 + 733320ν^2/75601)
+            + (v/c)^6 * (-265361599//33869248
+                         + 18177898147ν/321757856
+                         - 722521125ν^2/5745676
+                         + 261283995ν^3/2872838)
+        )
+    end
+
+    # Spin terms
+    if ℓₘₐₓ ≥ 0
+        Ψ_M[Yindex(0,0,ℓₘᵢₙ)] += Ψ_M₀ * @pn_expansion(
+            # Eq. (A1b) of MSB+ (2022)
+            + (v/c)^3 * (14//3 * Sₗ + 2δ * Σₗ) / M^2
+            + (v/c)^4 * ((-16S⃗·S⃗ + 3Σ⃗·Σ⃗ + 32Sₗ^2 + 9Σₗ^2)/12
```

Comment:
The minus sign on $16S\cdot S$ is suppose to be acting on the whole expression with $12M^{4}$ in parentheses